src/components/Aframe/arcball-camera.js

```diff
@@ -117,22 +120,22 @@ AFRAME.registerComponent('arcball-camera', {
       console.log("camera old pos")
       console.log(this.oldPosition);
       //el.getObject3D('camera').position.copy(this.oldPosition);
-      el.getObject3D('camera').position.set(0,0,2); 
+      el.getObject3D('camera').position.set(0,0,2);
       console.log("camera matrixWorld")
       console.log(el.getObject3D('camera').matrixWorld);
       console.log("target: ")
       console.log(this.controls.target);
       console.log("this.meshObjectHandler.matrixWorld");
       console.log(this.meshObjectHandler.matrixWorld);
       this.meshObjectHandler.position.set(0,0,0);
-        
+
       el.getObject3D('camera').matrixWorld.copy(this.oldMatrix);
       el.getObject3D('camera').updateMatrix();
       console.log("el.getObject3D('camera').matrixWorld")
       console.log(el.getObject3D('camera').matrixWorld);
       console.log("exit VR")
       //el.getObject3D('camera').position.set(0, 0, 0);
-      //if (el.hasAttribute('look-controls')) { 
+      //if (el.hasAttribute('look-controls')) {
       // }
       */
```

Comment:
Delete comments?

Reply:
ah, I didn't even look at this code - I touched elsewhere in the file and my editor added opinions... I'm going to defer deleting the code to @RobertGemmaJr

Reply:
I haven't touched this file so I would say it's safe to delete

Reply:
I thought from the other PR you were going to go through and do a pass of deleting all the comments, so this would roll into there

Reply:
Oh I just deleted the stuff from that one file. I can take up that task in  a new branch though